README.md

```diff
@@ -4,11 +4,13 @@
 |:--:|:--:|
 |[![Master Build Status](https://travis-ci.org/OmniSharp/omnisharp-vscode.svg?branch=master)](https://travis-ci.org/OmniSharp/omnisharp-vscode)|[![Dev Build Status](https://travis-ci.org/OmniSharp/omnisharp-vscode.svg?branch=dev)](https://travis-ci.org/OmniSharp/omnisharp-vscode)|
 
-Don't install this extension unless you want to try an early version. 
-
-This is a preview of C# support in Visual Studio code. It is designed to work well with the upcoming [.NET Core 1.0](http://dotnet.github.io/). 
+This is a preview of C# support in Visual Studio code. It is designed to work well with [.NET Core 1.0](http://dotnet.github.io/). 
 These new projects are based on project.json and the [.NET CLI](https://dotnet.github.io/getting-started/).
 
+Support for .csproj projects will be improved in future versions of this extension. For now, if you are targeting Mono scenarios (ex: Unity 
+or Xamarin) you may want to instead install the 'Built-in Omnisharp' extension. Note that the name of the Omnisharp extension is no longer 
```

Comment:
At least on my machine it is called 'Built-in Omnisharp' which is wrong since the extension isn't built in any longer. So I figured we should have some sort of clarifying comment.